performance/scripts/preScript.js

```diff
@@ -0,0 +1,39 @@
+const http = require('http');
+
+module.exports = async function (context, commands) {
+
+    const hostname = 'demo';
+    const port = '8080';
+    const path = 'demo/performance.html';
+
+    return new Promise(resolve => {
```

Comment:
Het begint nu wat groot en slechter leesbaar te worden. Misschien best nog opsplitsen in meerdere methodes?

Reply:
Beetje refactored. Feedback welkom.